py3xui/client/client.py

```diff
@@ -65,5 +67,5 @@ class Client(BaseModel):
     flow: str = ""
     limit_ip: int = Field(default=0, alias=ClientFields.LIMIT_IP)  # type: ignore
     sub_id: str = Field(default="", alias=ClientFields.SUB_ID)  # type: ignore
-    tg_id: str = Field(default="", alias=ClientFields.TG_ID)  # type: ignore
+    tg_id: Optional[int | str] = Field(default=None, alias=ClientFields.TG_ID)  # type: ignore
```

Comment:
Change to modern typehint int | str | None

Reply:
> int | str | None

Ok, but api dont return "None"